src/Lexer.js

```diff
@@ -36,11 +36,16 @@ import {LexerInterface, Token} from "./Token";
 const commentRegexString = "%[^\n]*[\n]";
 const controlWordRegexString = "\\\\[a-zA-Z@]+";
 const controlSymbolRegexString = "\\\\[^\uD800-\uDFFF]";
+const combiningDiacriticalMarkString = "[\u0300-\u036f]";
```

Comment:
This contains many more characters than we support.  Where do we generate an error if someone uses an unsupported combining mark?  Maybe add a test for this.

Reply:
I added an error test (in `error-spec.js` because I wanted `toFailWithParseError`). Incidentally, most of my tests are in `katex-spec.js` because I wanted `toParseLike`, but they would make more sense in `unicode-spec.js`...  Not sure how to best share code between these test modules.

Reply:
I've created #1021 to extract helper functions into test-utils.js.  I can do that this weekend and then we could rebase this with the tests in the proper suite.